nifi-nar-bundles/nifi-graph-bundle/nifi-graph-processors/src/main/java/org/apache/nifi/processors/graph/ExecuteGraphQueryRecord.java

```diff
@@ -223,14 +237,19 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
                     }
 
                     dynamicPropertyMap.putAll(input.getAttributes());
+                    if (getLogger().isDebugEnabled()) {
+                        getLogger().info("Parameter map: " + dynamicPropertyMap);
```

Comment:
Should this be changed to `getLogger().debug()`?  Using a parameterized logging statement with `dynamicPropertyMap` would also optimize the implementation:

```suggestion
                        getLogger().debug("Dynamic Properties: {}", new Object[]{dynamicPropertyMap});
```